README.md

```diff
@@ -1,2 +1,34 @@
-# octopus
-The test runner for acceptance tests
+# Octopus
+
+## Overview
+
+Octopus is a testing framework that allows running tests defined as Docker images on a running cluster.
+Think about Octopus as a `helm test` on steroids.
```

Comment:
Seriously? 🙂 🐙 

As much as I love the metaphor, I think we should skip it in the official README.

Reply:
It was the best sentence :(